mockgcp/mockcontainer/nodepool.go

```diff
@@ -203,13 +203,28 @@ func (s *ClusterManagerV1) populateAutoprovisioningNodePoolDefaults(obj *pb.Auto
 			"https://www.googleapis.com/auth/cloud-platform",
 		}
 	} else {
+		// Reach out to API team to understand why when the input is
```

Comment:
maybe let's create an issue and link it in here?

Reply:
Created an internal ticket as we'll probably need to reach out to the API team internally, so didn't link it here.